src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -177,7 +178,7 @@ public void onResume() {
     reloadTimer.scheduleAtFixedRate(new TimerTask() {
       @Override
       public void run() {
-        Util.runOnMain(() -> loadChatlist());
+        Util.runOnMain(() -> { list.getAdapter().notifyDataSetChanged(); });
```

Comment:
this just reverts a wrong change, see https://github.com/deltachat/deltachat-android/pull/1975#pullrequestreview-719498036